pkg/models/registries/registries.go

```diff
@@ -68,6 +70,23 @@ func RegistryVerify(authInfo AuthInfo) error {
 		ServerAddress: authInfo.ServerHost,
 	}
 
+	// Sometimes request DockerHub V2 API will timeout in China. Skipped.
```

Comment:
It would be nice if you could add some test cases.
like 
`docker.io "" ""`
`dockerhub.qingcloud.com guest guest`

Reply:
added some test registry that can login with guest.